### PR TITLE
Add CFP & Tickets links for 2022

### PR DESCRIPTION
**Pull request by abelsromero**

### CODE REVIEW

File: 2022/_data/homepage.json [hunk 1]
```diff
@@ -57,7 +57,7 @@
   "days": "18-19-20",
   "from-to": "18th to 20th of July",
   "from_to_days": "18th to 20th of July 2022",
-  "cfp_closing": "18th of June (last day)",
+  "cfp_closing": "18th of May (last day)",
```
Comment (abelsromero) [inline 2022/_data/homepage.json:60]: TBC

**Review by anyulled — APPROVED** [review]

File: 2022/_data/homepage.json [hunk 2]
```diff
@@ -14,7 +14,7 @@
   "ticket_soon": "SOON",
   "ticket_soldout": "SOLD OUT",
   "ticket_staytuned" : "Stay tuned !!",
-  "tickets_url": "https://hopin.com/events/jbcnconf-2021-virtual/",
+  "tickets_url": "https://tickets.jbcnconf.com/event/2022",
```
Comment (icougil) [inline 2022/_data/homepage.json:17]: The main URL should work (once we publish the event):
```suggestion
  "tickets_url": "https://tickets.jbcnconf.com/",
```

File: 2022/_data/homepage.json (hunk 1 above)
Comment (icougil) [inline 2022/_data/homepage.json:60]:
```suggestion
  "cfp_closing": "30th of April (last day)",
```

**Review by icougil — CHANGES_REQUESTED** [review]
Some suggestions

**Pull request merged**